scripts/gtcadis.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+import argparse
+import ConfigParser
+
+import numpy as np
+from pyne.mesh import Mesh
+from pyne.partisn import write_partisn_input, isotropic_vol_source
+from pyne.dagmc import discretize_geom, load
+from pyne import nucname
+from pyne.bins import pointwise_collapse
+
+
+config_filename = 'config.ini'
+
+config = \
+    """
+# Optional step to assess all materials in geometry for compatibility with
+# SNILB criteria
+[step0]
+
+# Prepare PARTISN input file for adjoint photon transport
+[step1]
+# Path to hdf5 geometry file for photon transport
+geom_file:
+# Volume ID of adjoint photon source cell on
+# DAGMC input [Trelis/Cubit .sat file]
+src_cell:
+# Volume [cm^3] of source cell (detector)
+src_vol:
+# Define mesh that covers entire geometry:
+# xmin, xmax, number of intervals
+xmesh:
```

Comment:
replace with `origin`, `[x|y|z]mesh` and `[x|y|z]ints` to be similar to MCNP structure (or fold the origin into the *mesh?)